src/core_plugins/timelion/public/directives/expression_directive.html

```diff
@@ -1,23 +1,14 @@
-<div class="timelionSearchInputContainer">
-  <input
-    id="timelionSearchInput"
-    input-focus
-    ng-model="sheet"
-    timelion-expression="{{sheet}}"
-    placeholder="Expression..."
-    aria-label="Expression input"
```

Comment:
It's still here, just moved to https://github.com/elastic/kibana/pull/11638/files#diff-c2e6e1cb6f3958d224cd3f1ff164a4f9R35